README.md

```diff
@@ -371,8 +371,9 @@ Carbon focused talks from the community:
 
 -   [The Carbon Language: Road to 0.1](https://chandlerc.blog/slides/2024-ndc-techtown-carbon-road-to-0-dot-1),
     NDC {TechTown}
--   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://chandlerc.blog/slides/2024-cppnorth-design-stories),
+-   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://www.youtube.com/watch?v=8SGMy9ENGz8),
     CppNorth
+    ([slides](https://chandlerc.blog/slides/2024-cppnorth-design-stories))
```

Comment:
To note as I did on #4576, there was a decision on #3158 not to keep slides for each talk. If you're going to change the format, please add slides for all talks below as well. However, I will suggest adopting the format suggested on #3158 if that's the path taken. (`Talk name, con ([video](youtube link), [slides](slides link))`)

You will probably need to research through historical talks to find slides.

Reply:
FWIW, I think it would be good to add slide links for other talks where we have both a video and slides, but it seems reasonable to me to do that in follow-up PRs.

However, I think the bigger thing is that it'd be good to be consistent about which links are videos and which are slides. Mostly this is because I think it could surprise folks clicking and expecting slides but to start getting audio from a video playing. (Maybe I'm the only one who routinely startles themselves with audio from a website accidentally?)

I could see two formats working here @josh11b:

```
- [Title](video-link) ([slides](slides-link))
- Title (no video yet, [slides](slides-link))
```

Or:

```
- Title ([video](video-link), [slides](slides-link))
- Title (no video yet, [slides](slides-link))
```

No strong preference between these. Happy if others have a preference to pick that one?